connect/connect-core/connect-worker/worker-api/src/main/java/io/miragon/miranum/connect/worker/impl/WorkerInfoRegistry.java

```diff
@@ -0,0 +1,17 @@
+package io.miragon.miranum.connect.worker.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class WorkerInfoRegistry {
+
+    private final List<WorkerInfo> workerInfos = new ArrayList<>();
+
+    public void addWorkerInfo(WorkerInfo workerInfo) {
+        workerInfos.add(workerInfo);
```

Comment:
I think the WorkerRegistry is something the WorkerExecuterApi in the PR of @lmoesle should use.

In my point of view we should first merge the PR of @lmoesle.